wgs.yaml

```diff
@@ -56,6 +56,67 @@ workinggroups:
   - name: katib
     owners:
     - https://raw.githubusercontent.com/kubeflow/katib/master/OWNERS
+- dir: wg-deployment
+  name: Deployment
+  mission_statement: >
+    WG Deployment covers developing, maintaining artifacts to deploy and operate Kubeflow.
+
+  charter_link: charter.md
+  label: area/wg-deployment
+  leadership:
+    chairs:
+    - github: Jeffwan
+      name: Jiaxin Shan
+      company: AWS
+    - github: animeshsingh
+      name: Animesh Singh
+      company: IBM
+    - github: swiftdiaries
+      name: Adhita Selvaraj
+      company: Cisco
+    - github: yanniszark
+      name: Yannis Zarkadas
+      company: Arrikto
+    tech_leads:
+    - github: Jeffwan
+      name: Jiaxin Shan
+      company: AWS
+    - github: animeshsingh
+      name: Animesh Singh
+      company: IBM
+    - github: yanniszark
+      name: Yannis Zarkadas
+      company: Arrikto
+  meetings:
+  - description: Regular WG Meeting (Pacific AM)
+    day: Tuesday
+    time: "08:00"
+    tz: PT (Pacific Time)
+    frequency: biweekly - every other Tuesday
+    archive_url: http://bit.ly/kf-meeting-notes
```

Comment:
Will the meeting notes be the same as the current community meet?

Reply:
Yeah, at least in the beginning it will be